site/en/docs/extensions/mv3/getstarted/tut-focus-mode/index.md

```diff
@@ -252,8 +252,8 @@ Just for fun, let's add a shortcut to make it easier to enable or disable focus
   "commands": {
     "_execute_action": {
       "suggested_key": {
-        "default": "Ctrl+U",
-        "mac": "Command+U"
+        "default": "Ctrl+Q",
+        "mac": "Command+Q"
```

Comment:
@ambujsahu81 I'm curious about the benefits of switching to a different shortcut 🤔

Reply:
@AmySteam  thanks for the review 🙂. I have updated the shortcut because when i used the `CTRL + U` in window 10 in chrome it opens non-editable HTML source code of the page instead of executing the extension code.

Reply:
Interesting! I only tested this on a mac, so thanks for bringing this up as a windows user! 🙂👍
My only hesitation is that Cmd + Q is the shortcut that quits the active program/application. 

@ambujsahu81 Can you suggest a different shortcut compatible with Windows and macOS? I appreciate your help on this! 😃

Reply:
@AmySteam  Thanks for explaining the issue with the shortcut `Cmd + Q` .  I have used this [Chrome keyboard shortcuts link ](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Cwebpage-shortcuts) to check reserved keys for both mac and (windows/Linux) OS, it seems `CTRL + B` & `Cmd + B` might be good option to use.

Reply:
Okay, let's go with `CTRL + B` and `Cmd + B`! 

Can you also make a PR to change the shortcut on the [GitHub example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/tutorials/focus-mode)? 
Thank would be amazing 😀

Reply:
Sorry for the delay, yes i am creating a pr for the github example. 🙂

Reply:
Thanks @ambujsahu81! 💯